### PR TITLE
feat(www): adds faq page

### PR DESCRIPTION
**Pull request by ixahmedxi**
## 🎯 Changes

This PR creates a FAQ page that renders a list of common questions and their answers.

## ✅ Checklist

- [x] I have followed the steps listed in the [Contributing guide](https://github.com/trpc/trpc/blob/main/CONTRIBUTING.md).
- [x] If necessary, I have added documentation related to the changes made.
- [x] I have added or updated the tests related to the changes made.

### CODE REVIEW

**Comment by juliusmarminge** [comment]
image

Alignment looks a bit off but otherwise I like it

**Review by KATT — COMMENTED** [review]
Can we add structured data JSON thingie? https://developers.google.com/search/docs/appearance/structured-data/faqpage

File: www/src/pages/faq.tsx [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+        <p className="pt-2 text-center text-zinc-700 dark:text-zinc-300">
+          A collection of the most commonly asked questions in regard to
+          everything tRPC.
+        </p>
+        <div className="mt-6">
+          {data.map((item, index) => (
+            <Disclosure
+              as="div"
+              key={`item-${index}`}
+              className="max-w-3xl mx-auto mt-3"
```
Comment (juliusmarminge) [inline www/src/pages/faq.tsx:77]: Align text to left here? Centered looks a bit odd

image

Comment (ixahmedxi) [reply www/src/pages/faq.tsx:77]: yeah that's true, ill change that ❤️

File: www/static/robots.txt [hunk 2]
```diff
@@ -0,0 +1,2 @@
+User-agent: *
+Disallow:
```
Comment (KATT) [inline www/static/robots.txt:2]: why?

File: www/versioned_docs/version-9.x/main/faq.mdx [hunk 3]
```diff
@@ -0,0 +1,10 @@
+---
+id: faq
+title: Frequently Asked Questions
+sidebar_label: Frequently Asked Questions
+slug: /faq
+---
+
+import Content from '@site/unversioned/_faq.mdx';
+
+<Content />
```
Comment (KATT) [inline www/versioned_docs/version-9.x/main/faq.mdx:10]: Let's get rid of v9 support

File: www/versioned_sidebars/version-9.x-sidebars.json [hunk 4]
```diff
@@ -8,6 +8,7 @@
         "main/introduction",
         "main/quickstart",
         "main/example-apps",
+        "main/faq",
```
Comment (KATT) [inline www/versioned_sidebars/version-9.x-sidebars.json:11]: Let's get rid of v9 support

**Review by KATT — CHANGES_REQUESTED** [review]
@ixahmedxi you reckon we could get this up to date?


- [ ] Remove it from v9 docs - we don't need it
- [ ] Make sure that each answer is easy to define
- [ ] Make sure we can have links in the reply -- ideally markdown?



maybe a struct like

```ts
interface FAQ {
  q: string;
  a: string;
}

const items: FAQ[] = [
  {
    q: "Do you like cats",
    a: "Yes, cats are **great**",
  }
]
```

**Comment by ixahmedxi** [comment]
@KATT could definitely get this up to date, this weekend I should have some free time and will tackle it :rocket:

**Pull request closed without merging**